config/v1/types_image.go

```diff
@@ -37,6 +37,23 @@ type Image struct {
 	Status ImageStatus `json:"status"`
 }
 
+// ImportModeType describes how to import an image manifest.
+// +enum
+// +kubebuilder:validation:Enum:=Legacy;PreserveOriginal
+type ImportModeType string
+
+const (
+       // ImportModeLegacy indicates that the legacy behaviour should be used.
+       // For manifest lists, the legacy behaviour will discard the manifest list and import a single
+       // sub-manifest. In this case, the platform is chosen in the following order of priority:
+       // 1. tag annotations; 2. control plane arch/os; 3. linux/amd64; 4. the first manifest in the list.
+       // This mode is the default.
+       ImportModeLegacy ImportModeType = "Legacy"
+       // ImportModePreserveOriginal indicates that the original manifest will be preserved.
+       // For manifest lists, the manifest list and all its sub-manifests will be imported.
+       ImportModePreserveOriginal ImportModeType = "PreserveOriginal"
```

Comment:
This is good, we need to include this in the field description as well so end users will get this when they describe the field